modules/visual_script/doc_classes/VisualScript.xml

```diff
@@ -26,23 +26,24 @@
 			</return>
 			<argument index="0" name="name" type="StringName">
 			</argument>
+			<argument index="1" name="func_node_id" type="int">
```

Comment:
You need to set the name in the C++ `bind_method`. This field is generated and will be overridden with `arg1` again.